test/hotel_test.rb

```diff
@@ -0,0 +1,282 @@
+
+
+
+require_relative 'test_helper'
+
+
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe "initialize" do 
+  it "I can access the list of all of the rooms in the hotel" do 
+    hotel = HotelSystem::Hotel.new 
+    hotel.must_be_instance_of HotelSystem::Hotel
+  end 
+  it "an element in Hotel object must be a room" do 
+    hotel = HotelSystem::Hotel.new 
+    room = hotel.rooms[1]
+    room.must_be_instance_of HotelSystem::Room 
+
+  end 
+
+end
+
+
+describe "other instance methods" do 
+
+  it "finds a room " do 
+    hotel = HotelSystem::Hotel.new
+    room = hotel.find_room(1)
+    room.must_be_instance_of HotelSystem::Room
+
+  end 
+
+  it "finds an array of reservations" do 
+  
+    hotel = HotelSystem::Hotel.new
+
+    room = hotel.find_room(1)
+    
+    res1 = HotelSystem::Reservation.new(1, Date.today, Date.new(2020,03,29))
```

Comment:
This test will break after March 29th 2020:

```suggestion
    res1 = HotelSystem::Reservation.new(1, Date.today, Date.today + 15)
```